app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::GreetingsController < ApiController
+  def index
```

Comment:
- [ ] Both are rails and react (api/v1/greetings#index) are displaying blank screen and empty array on their respective routes. Kindly check and improve upon the same.

Reply:
Thanks for your time Sir. It is highly appreciated. 
However. Can you please tell me how are you running the app? I can't replicate the behavior that you describe. 
I keep trying different things but the app always works in my end. I have recorded a video showing how it does.

[![Watch the video](https://i.imgur.com/vKb2F1B.png)](https://drive.google.com/file/d/1V7lvLX1NN473Ylj-qIh8gRLpWiYzeZmc/view?usp=sharing)

Reply:
Hello @Metsanpeitto 
As discussed, this issue was due to your application running on a different environment. From a recheck, **your app is running correctly!**

Reply:
Ok @rloterh . Thanks for your time !